istio-telemetry/kiali/templates/configmap.yaml

```diff
@@ -8,15 +8,20 @@ metadata:
     release: {{ .Release.Name }}
 data:
   config.yaml: |
-    istio_namespace: {{ .Release.Namespace }}
+    istio_component_namespaces:
+      grafana: {{ .Values.global.telemetryNamespace }}
+      tracing: {{ .Values.global.telemetryNamespace }}
+      pilot: {{ .Values.global.configNamespace }}
+      prometheus: {{ .Values.global.prometheusNamespace }}
+    istio_namespace: {{ .Values.global.istioNamespace }}
```

Comment:
@jmazzitelli, looks like you coordinated this change on Kiali's side. Does this look correct to you?

Reply:
@ayj yes, that all looks correct.

cc @jshaughn -- just asking for another set of eyes, he did some work on the kiali server to also deal with these settings.

Reply:
This looks correct to me.  I question whether we may want the policy namespace in Kiali as well, but for now it seems we don't need it.  It would be great for someone to vet what we've done by doing a multi-namespace istio deployment and trying to look at Istio topology and definitions in Kiali.

Reply:
I am looking at this part. I am not sure what you want to vet. follow the installer document, we can setup a multi-namespace istio.

Reply:
@clyang82 I was just hoping for you, or someone outside of this team, to try running Kiali with a multi-namespace Istio install.  Just for an external validation of the changes we made to support that scenario.

Reply:
Yes. I did tried before. it was working with a multi-namespace Istio install. Thanks

Reply:
Good news, thanks!